analytics/analytics.go

```diff
@@ -0,0 +1,258 @@
+package analytics
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/alecthomas/kong"
+	"github.com/denisbrodbeck/machineid"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/user"
+	"path/filepath"
+	"strings"
+	"time"
```

Comment:
(nitpick) please, separate the stdlib dependencies and the third party ones with an empty line.

Reply:
Happy to, but `go fmt` rewrites it to what's committed.

Reply:
Fixed in e96d5d3d756311a7dfcc281fc6fc1a361a2162f2.